synapse/config/experimental.py

```diff
@@ -29,9 +29,26 @@ def read_config(self, config: JsonDict, **kwargs):
         self.msc2858_enabled = experimental.get("msc2858_enabled", False)  # type: bool
 
         # Spaces (MSC1772, MSC2946, MSC3083, etc)
-        self.spaces_enabled = experimental.get("spaces_enabled", False)  # type: bool
+        self.spaces_enabled = experimental.get("spaces_enabled", True)  # type: bool
         if self.spaces_enabled:
             KNOWN_ROOM_VERSIONS[RoomVersions.MSC3083.identifier] = RoomVersions.MSC3083
 
         # MSC3026 (busy presence state)
         self.msc3026_enabled = experimental.get("msc3026_enabled", False)  # type: bool
+
+    def generate_config_section(self, **kwargs):
+        return """\
```

Comment:
We may want to leave a note here stating that we only include experimental features that are on by default in the sample config.

Reply:
Is that true though? This only includes the one enabled by default, but I'm unsure that's the *reason* it is the only one included.

Reply:
I'm going to merge this as is for now, we can update it when (if?) we add more here.